src/stylesheets/elements/_tags.scss

```diff
@@ -12,8 +12,7 @@
   }
 }
 
-.usa-label-big {
-  @extend .usa-label;
```

Comment:
I see this use of an `@extend` here — are we consciously avoiding this technique now in favor of requiring both classes?